package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "glob-auto-library",
-	"version": "1.1.11",
+	"version": "1.1.13",
```

Comment:
Current version on https://www.npmjs.com/package/glob-auto-library 1.1.14 - please update to 1.1.15